lib/widgets/drive.coffee

```diff
@@ -51,18 +50,38 @@ getDrives = ->
 # visuals.drive('Please select a drive').then (drive) ->
 # 	console.log(drive)
 ###
-module.exports = (message = 'Select a drive') ->
-	getDrives().then (drives) ->
 
-		if _.isEmpty(drives)
-			throw new Error('No available drives')
+module.exports = (message = 'Select a drive') ->
 
-		form.ask
-			type: 'list'
-			name: 'drive'
+	getDrives().then (drives) ->
+		options =
 			message: message
+			name: 'drives'
 			choices: _.map drives, (drive) ->
 				return {
 					name: "#{drive.device} (#{drive.size}) - #{drive.description}"
 					value: drive.device
 				}
+
+		ui = new UI
+			input: process.stdin
+			output: process.stdout
+		list = new InquirerList(options, ui.rl)
+		scanner = new DriveScanner(getDrives, interval: 5000, drives: drives)
+
+		scanner.on 'add', (drive) ->
+			list.opt.choices.push
+				name: "#{drive.device} (#{drive.size}) - #{drive.description}"
+				value: drive.device
+			list.render()
+
+		scanner.on 'remove', (drive) ->
+			list.opt.choices.choices = _.reject list.opt.choices.choices, (previousDrive) ->
```

Comment:
You're duplicating the `_.reject` predicate here and below. You can extract to another function and reuse.